server/channels/jobs/jobs.go

```diff
@@ -56,8 +69,6 @@ func (srv *JobServer) _createJob(c *request.Context, jobType string, jobData map
 		Data:     jobData,
 	}
 
-	job.InitLogger(c.Logger())
```

Comment:
This line was the reason that I went with the approach to embed the logger in the first place. I wanted to pass the logger from the `request.Context` all the way down to the worker. That makes it easier for admins to distinguish between manual runs and automatic runs.

But ultimately, that doesn't even work with the code today as new jobs first get written to the DB, where the logger is lost. Hence, I'm fine with removing the logger from `model.Job`.

I'm still wondering if we can make it more clear for admins which jobs are manually triggered. But that can be done in a separate PR.